### PR TITLE
Feat/UI cleanup

### PR DESCRIPTION
**Pull request by LukeFinch**
- UI Improvements for second screen view
- New Icons for flow & second screen
- Language moved to settings
- Reduced copy for session recording information

copilot:summary

### CODE REVIEW

**Comment by LukeFinch** [comment]
Some translations are still missing before this is ready to merge

File: .env.production.example [hunk 1]
```diff
@@ -2,4 +2,4 @@ MIXPANEL_ACCESS_TOKEN=
 STORYBLOK_ACCESS_TOKEN=
 ENVIRONMENT=production
 LICENSE_API_URL=https://licence.tokens.studio
-LAUNCHDARKLY_SDK_CLIENT=626fb05d52e5c715abd11b5e
\ No newline at end of file
+LAUNCHDARKLY_SDK_CLIENT=626fb05d52e5c715abd11b5
```
Comment (six7) [inline .env.production.example:5]: This seems like a mistake

**Comment by six7** [comment]
@LukeFinch can we get this one merged today or tomorrow? would love to ship those UI updates 🙏

**Comment by LukeFinch** [comment]
> @LukeFinch can we get this one merged today or tomorrow? would love to ship those UI updates 🙏

On it :)

File: src/app/components/SecondScreen/index.tsx [hunk 2]
```diff
@@ -62,86 +68,95 @@ export default function SecondScreen() {
...
+          <Heading>
+            Second Screen
+            {' '}
+            <StyledBetaBadge>BETA</StyledBetaBadge>
+          </Heading>
           <StyledP>
-            {t('secondScreenIsInBetaSomeFeaturesMayNotWorkAsExpected')}
+            {t('secondScreenExplainer')}
           </StyledP>
+          <StyledReadMoreLink href="docs.tokens.studio" target="_blank" rel="noreferrer">{t('readMore', { ns: 'general' })}</StyledReadMoreLink>
```
Comment (six7) [inline src/app/components/SecondScreen/index.tsx:83]: can we make that a `https://...`?

**Pull request closed without merging**